aws/codedeploy_application.go

```diff
@@ -14,40 +10,39 @@ import (
 	"github.com/gruntwork-io/go-commons/errors"
 	commonTelemetry "github.com/gruntwork-io/go-commons/telemetry"
 	"github.com/hashicorp/go-multierror"
+	"sync"
 )
 
-func getAllCodeDeployApplications(session *session.Session, excludeAfter time.Time, configObj config.Config) ([]string, error) {
-	svc := codedeploy.New(session)
-
+func (c CodeDeployApplications) getAll(configObj config.Config) ([]*string, error) {
```

Comment:
Same shorthand conversation here, perhaps `cda`? I feel like single letter abbreviations, while being golang cannon, make a some things more difficult:
- Find & Replace
- Regex searches
- Function walking
- Refactoring

Reply:
Ah definitely, In order to prevent back-and-forth approval again, I'll go through consistency related changes at once after all refactoring since it would be quite easy to make!